BUILDING.md

```diff
@@ -0,0 +1,65 @@
+# Building gamerOS
+
+This document contains instructions on how to build, install and ship a new gamerOS image.
+
+# Build requirements
+
+Building the gamerOS image can currently only be done on **Arch Linux**.
+
+The following packages will need to be installed to be able to build the gamerOS image:
+- arch-install-scripts
+- btrfs-progs
+- [pikaur](https://aur.archlinux.org/packages/pikaur/) (not available in the official repo)
```

Comment:
(note to self) also need to add the multib repo!

Reply:
I added this to the requirements.